lib/x11utils.pm

```diff
@@ -322,4 +323,20 @@ sub untick_welcome_on_next_startup {
     }
 }
 
+=head2 workaround_boo1170586
+
+ workaround_boo1170586();
+
+Kill broken opensuse-welcome window and restart it properly to workaround boo#1170586.
+
+=cut
+sub workaround_boo1170586 {
+    record_soft_failure('opensuse-welcome auto-start is broken, boo#1170586');
+    wait_screen_change { send_key desktop_runner_hotkey };
+    type_string "killall /usr/bin/opensuse-welcome\n";
+    wait_screen_change { send_key desktop_runner_hotkey };
+    type_string "opensuse-welcome\n";
```

Comment:
this is not how the desktop runner is usually called and likely to fail. I suggest to use either `start_x11_program` or you do what https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/6baaf86bc5ca2754bf5649474e5bcdcd6621149c does. Unfortunately again that commit is an example of "stating the obvious" rather than explaining the reasoning or the benefit of the commit but in short: It is meant to terminate a program reliably when previously using the desktop runner showed instabilities

Reply:
At least it passed right now on aarch64. The instability was related to KDE and maybe GNOME is fine? I would be in favor to give it a try.

Reply:
Converted to `x11_start_program`. Let's see how it goes.